#!/usr/bin/env python
PACKAGE = "stroll_bearnav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("currentTime", int_t, 0, "time of rosbag occurence",    3,     0, 20)


exit(gen.generate(PACKAGE, "stroll_bearnav", "listener"))

